ui/src/store/features/github/async-thunk/login.ts

```diff
@@ -45,6 +45,7 @@ export const login = createAsyncThunk(
         throw Error('Invalid response type');
       }
     } catch (error) {
+      AppLog.error(error);

```

Comment:
```suggestion
      AppLog.errorToast('Github login failed. Please try again later.', error);
```
and remove the next line